addon/utils/ajax.ts

```diff
@@ -1,6 +1,6 @@
-import jQuery from 'jquery';
+import * as jQuery from 'jquery';
 
-const ajax =
+const ajax: typeof jQuery.ajax =
```

Comment:
because `FastBoot.require` is not well-typed, the type of `ajax` ended up being intersected with `any` (resulting in a type of `any` because `X | any` is `any` regardless of X)